src/dict/dsl.cc

```diff
@@ -1042,6 +1042,8 @@ QString const & DslDictionary::getDescription()
     QString data, str;
 
     str = annStream.readLine();
+    auto encoding = Utils::detectEncoding( str );
+    annStream.setEncoding( encoding );
```

Comment:
This does not fix the problem.

Using QStringConverter to detect encoding is the default behaviour of QTextStream already.

Reply:
> Using QStringConverter to detect encoding is the default behaviour of QTextStream already.

are you sure?

I know that the QTextStream use BOM to detect encoding.

but QStringConverter::encodingForData is not only based on BOM mark.

Reply:
> Using QStringConverter to detect encoding is the default behaviour of QTextStream already.

Yes ,this is true. confirmed.

But still have some space ,if the first character has been provided if possible.